### DISCUSSION TITLE
Update-ServiceNowRecord : Update model

### DISCUSSION THREAD

**Discussion by GeekArt-Syn** [Snow-Shell/servicenow-powershell#181] [Q&A] [answered]
Hi !
Firstly, many thanks for this module, it is very useful to me 👍 

Although I've got a question :
I'm unable to update the model value of an asset in the table alm_hardware :
Using :
`Update-ServiceNowRecord -Table alm_hardware -Id "123456789123456789" -Values @{model = "HP EliteBook 840 G3"} -PassThru`
The record is not updated :(

I've tried to update the model using the corresponding sys_id to no avail :
`Update-ServiceNowRecord -Table alm_hardware -Id "123456789123456789" -Values @{model =  "d26a148db232399619e"} -PassThru`

When I try to update the "install_status" field, it does work as expected :
`Update-ServiceNowRecord -Table alm_hardware -Id "123456789123456789" -Values @{install_status = "1"} -PassThru`

Any clue on what I am doing wrong ?

**Comment by gdbarron** [comment, marked as answer]
Hi @GeekArt-Syn, I'm seeing the same behavior.  Have you tried changing this via the gui?  Perhaps changing the model has an associated field that needs to change with it?

**Reply by GeekArt-Syn** [reply]
I can confirm it is the same when using the gui : The field is greyed out.
I guess the behavior it is either related to how the system was designed or it is due to my account not having the necessary privileges.
Thanks anyway for looking into it :)
